.gitignore

```diff
@@ -1 +1,3 @@
-node_modules/
\ No newline at end of file
+.vscode/*
```

Comment:
I've never encountered this folder (.vscode/). What is it and what does excluding it via .gitignore amount to?

Reply:
config folder from visual code (custom settings can be kept here and applied to vscode per project). I just reverted this file.